app/views/legislation/questions/_votes.html.erb

```diff
@@ -0,0 +1,79 @@
+<% voted_classes = css_classes_for_vote(@legislation_question_votes, @question) %>
+<div id="<%= dom_id(@question)%>_votes" class="votes">
+  <% if user_signed_in? && current_user.level_two_verified? %>
+    <span class="in-favor">
+      <% if can?(:vote, question) %>
+        <%= link_to vote_legislation_process_question_path(question.process, question, value: "yes"),
+          method: "post", remote: true, class: "like #{voted_classes[:in_favor]}", title: t("legislation.questions.show.clear") do %>
```

Comment:
```suggestion
          method: "post", remote: true, class: "like #{voted_classes[:in_favor]}" do %>
```

IMHO adding a title with the same text as the link is unnecessary noise :thinking:. And, yes, I know it's done this way in other places :relieved:.